Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rrt"
-version = "0.3.0"
+version = "0.5.0"
```

Comment:
`main` branch is 0.3.0, but it seems 0.4.0 is already released: https://crates.io/crates/rrt/0.4.0